tools/editor/multi_node_edit.cpp

```diff
@@ -52,8 +52,16 @@ bool MultiNodeEdit::_set(const StringName& p_name, const Variant& p_value){
 		Node*n=es->get_node(E->get());
 		if (!n)
 			continue;
+		else if (p_value.get_type() == Variant::NODE_PATH) {
```

Comment:
I think you should remove the `else` from here though. (and maybe add an additional newline..)